src/iop/demosaic.c

```diff
@@ -3917,7 +4665,7 @@ static void demosaic_method_xtrans_callback(GtkWidget *combo, dt_iop_module_t *s
   p->demosaicing_method = dt_bauhaus_combobox_get(combo) | DEMOSAIC_XTRANS;
   if((p->demosaicing_method > (DT_IOP_DEMOSAIC_PASSTHROUGH_MONOCHROME | DEMOSAIC_XTRANS))
      || (p->demosaicing_method < DT_IOP_DEMOSAIC_VNG))
-    p->demosaicing_method = DT_IOP_DEMOSAIC_MARKESTEIJN;
+    p->demosaicing_method = DT_IOP_DEMOSAIC_FDC;
```

Comment:
I guess this is clipping, if somehow we have a too big method id.
Still, i think it is better to fallback to old good `DT_IOP_DEMOSAIC_MARKESTEIJN`.